src/main/java/healthy/lifestyle/backend/workout/service/ExerciseServiceImpl.java

```diff
@@ -178,4 +178,10 @@ public List<ExerciseResponseDto> getDefaultExercises() {
                 })
                 .toList();
     }
+
+    @Override
+    @Transactional
+    public ExerciseResponseDto getDefaultExerciseById(long exerciseId) {
+        return modelMapper.map(exerciseRepository.findDefaultById(exerciseId), ExerciseResponseDto.class);
```

Comment:
Body parts and Http refs are stored in Exercise model as Set, so their order are not guaranteed. We need to ensure that each time exercise details are requested - it returns exercise details dto with same order of Body parts and Http refs.